src/api/public/apidocs-new/paths/attribute_namespace_attribute_name_meta.yaml

```diff
@@ -0,0 +1,199 @@
+get:
+  summary: Shows attribute.
+  description: Shows attribute.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/namespace.yaml'
+    - $ref: '../components/parameters/attribute_name.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/attribute.yaml'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      description: Not Found.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            namespace:
+              value:
+                code: not_found
+                summary: Couldn't find AttribNamespace
+              summary: Not Found (Namespace)
+            unknown_attribute:
+              value:
+                code: unknown_attribute
+                summary: "Unknown attribute 'OBS_TEST':'OwnerRootProjectTest'"
+              summary: Unknown Attribute
+  tags:
+    - Attributes
+
+post:
+  # 'post' and 'put' operations must be in sync
+  summary: Change attribute data. Create an attribute if it doesn't exist.
+  description: |
+    Change attribute data. Create an attribute if it doesn't exist.
```

Comment:
Please expand this description so it's understandable that this route is both for creating an attribute and updating it.

Reply:
Done.